vscode_extension/src/commands/toggleUntypedCodeHighlighting.ts

```diff
@@ -14,7 +13,13 @@ export async function toggleUntypedCodeHighlighting(
   context.log.info(
     `Untyped code highlighting: ${targetState ? "enabled" : "disabled"}`,
   );
-
-  await context.statusProvider.restartSorbet(RestartReason.CONFIG_CHANGE);
+  
+  context.statusProvider.activeLanguageClient?.sendNotification("workspace/didChangeConfiguration",
+    {
+      "settings": {
+        "highlightUntyped": targetState
+      }
+    }
+  );
```

Comment:
If there is no `context.statusProvider.activeLanguageClient` seems this whole flow would not make sense.

Could the implementation be changed so it checks/bails early if there is no `context.statusProvider.activeLanguageClient` ?

Something  like
```
const {activeLanguageClient: client} = context.statusProvider;
  if (!client) {
    context.log.debug("ToggleUntyped: No active Sorbet LSP.");
  }

…

  context.log.info(
    `ToggleUntyped: Untyped code highlighting: ${targetState ? "enabled" : "disabled"}`,
  );
  client.sendNotification("workspace/didChangeConfiguration", {
    settings: {
      highlightUntyped: targetState,
    },
  });
…
```